CHANGELOG.rst

```diff
@@ -17,5 +17,6 @@ master
 - (`#83 <https://github.com/openclimatedata/openscm/pull/83>`_) Rename OpenSCMDataFrame to ScmDataFrame
 - (`#78 <https://github.com/openclimatedata/openscm/pull/78>`_) Added OpenSCMDataFrame
 - (`#44 <https://github.com/openclimatedata/openscm/pull/44>`_) Add timeframes module
+- (`#41 <https://github.com/openclimatedata/openscm/pull/41>`_) Add DICE adapter
```

Comment:
Do we want to put this in order of time rather than PR number?

Reply:
To be honest I don't believe this list makes sense before the first release. But by number is good enough as the order per release section, I think. Otherwise, merging messes up the order easily, anyway.

Reply:
> Otherwise, merging messes up the order easily, anyway

Not if you're careful but it really doesn't matter